cosmos/bbn-test.json

```
@@ -26,6 +26,12 @@
       "coinMinimalDenom": "ubbn",
       "coinDecimals": 6,
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/bbn-test/chain.png"
+    },
+    {
+      "coinDenom": "eBABY",
+      "coinMinimalDenom": "ebbn",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/ohadbachner/keplr-chain-registry/main/images/bbn-test/eBABY.png"
```

```suggestion
      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/bbn-test/eBABY.png"
``` 
this would be correct !